packages/dataviews/src/dataviews-layouts/list/index.tsx

```diff
@@ -196,116 +233,60 @@ function ListItem< Item >( {
 				'is-selected': isSelected,
 				'is-hovered': isHovered,
 			} ) }
-			onMouseEnter={ handleMouseEnter }
-			onMouseLeave={ handleMouseLeave }
+			onMouseEnter={ handleHover }
+			onMouseLeave={ handleHover }
 		>
-			<HStack
-				className="dataviews-view-list__item-wrapper"
-				alignment="center"
-				spacing={ 0 }
-			>
+			<HStack className="dataviews-view-list__item-wrapper" spacing={ 0 }>
 				<div role="gridcell">
 					<Composite.Item
-						render={ <div /> }
-						role="button"
 						id={ generateItemWrapperCompositeId( idPrefix ) }
 						aria-pressed={ isSelected }
 						aria-labelledby={ labelId }
 						aria-describedby={ descriptionId }
 						className="dataviews-view-list__item"
 						onClick={ () => onSelect( item ) }
+					/>
```

Comment:
Yeah, this the bit I wondered most about but hoped the existing `aria-labeledby` and `aria-describedby` makes it okay.

> move all cell content (except for the actions buttons) inside it

The actions have to be a sibling of the title for the truncation to "just work". So both would have to stay outside the main button, but then they’d not be able to impact the layout of other content. Not to say that can’t be made to work, but then the title is outside the button anyway.

Reply:
I did some testing with VoiceOver and it announces the same content as on trunk when navigating the rows and gridcells here. I suppose to be totally sure testing with other screen readers is needed but I expect they all do well as support for `aria-label` ought to be good.

Reply:
@afercia maybe you could advise if this change is OK in terms of accessibility, or if it has unwanted consequences?

Reply:
@ciampo Technically, it's not that different from an icon-button where the SVG icon is `aria-hidden` so that the button is 'empty'. However, it's always preferable to [not use ARIA when possible](https://www.w3.org/TR/using-aria/#rule1). We shouldn't assume all assistive technologies understand ARIA, for example with some combination of browsers and speech recognition software. Is it possible to use visually hidden content instead of aria-label?
Anyways, I'm not sure this is the only problem with the list view implementation and the dataviews in general as an in-depth accessibility audit has still to happen and that's something I can't do alone.